data/card_definitions.json

```diff
@@ -22915,6 +22915,363 @@
 	},
 
 
+	{
+		"NOTE": "VEGA CARDS START",
+		"id": "vega_bloodyhighclaw",
+		"type": "ultra",
+		"display_name": "Bloody High Claw",
+		"force_cost": 0,
+		"gauge_cost": 3,
+		"range_min": 1,
+		"range_max": 2,
+		"speed": 6,
+		"power": 4,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "before",
+				"condition": "min_spaces_behind_opponent",
+				"condition_amount": 3,
+				"effect_type": "powerup",
+				"amount": 3
+			},
+			{
+				"timing": "hit",
+				"effect_type": "move_to_any_space"
+			}
+		],
+		"boost": {
+			"boost_type": "continuous",
+			"force_cost": 1,
+			"cancel_cost": -1,
+			"display_name": "Claw",
+			"effects": [
+				{
+					"timing": "set_strike",
+					"effect_type": "critical",
+					"character_effect": true
+				},
+				{
+					"timing": "during_strike",
+					"effect_type": "rangeup",
+					"amount": 0,
+					"amount2": 1
+				},
+				{
+					"timing": "cleanup",
+					"condition": "not_stunned",
+					"effect_type": "force_for_effect",
+					"per_force_effect": null,
+					"overall_effect": {
+						"effect_type": "sustain_this",
+						"card_name": "Bloody High Claw"
+					},
+					"force_max": 1
+				}
+			]
+		}
+	},
+	{
+		"id": "vega_splendidclaw",
+		"type": "ultra",
+		"display_name": "Splendid Claw",
+		"force_cost": 0,
+		"gauge_cost": 3,
+		"range_min": 1,
+		"range_max": 2,
+		"speed": 4,
+		"power": 7,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "before",
+				"effect_type": "close",
+				"amount": 3
+			},
+			{
+				"timing": "hit",
+				"effect_type": "opponent_discard_random",
+				"amount": 3
+			}
+		],
+		"boost": {
+			"boost_type": "continuous",
+			"force_cost": 0,
+			"cancel_cost": -1,
+			"display_name": "Mask",
+			"effects": [
+				{
+					"timing": "during_strike",
+					"effect_type": "powerup",
+					"amount": 1,
+					"and": {
+						"effect_type": "armorup",
+						"amount": 1
+					}
+				},
+				{
+					"timing": "cleanup",
+					"condition": "not_stunned",
+					"effect_type": "force_for_effect",
+					"per_force_effect": null,
+					"overall_effect": {
+						"effect_type": "sustain_this",
+						"card_name": "Splendid Claw"
+					},
+					"force_max": 1
+				}
+			]
+		}
+	},
+	{
+		"id": "vega_scarletterror",
+		"type": "special",
+		"display_name": "Scarlet Terror",
+		"force_cost": 0,
+		"gauge_cost": 0,
+		"range_min": 1,
+		"range_max": 2,
+		"speed": 6,
+		"power": 3,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "during_strike",
+				"condition": "is_critical",
+				"effect_type": "speedup",
+				"amount": 2
+			},
+			{
+				"timing": "before",
+				"effect_type": "close",
+				"amount": 2
+			}
+		],
+		"boost": {
+			"boost_type": "immediate",
+			"force_cost": 2,
+			"cancel_cost": -1,
+			"display_name": "Repel Leap",
+			"effects": [
+				{
+					"timing": "immediate",
+					"condition": "at_edge_of_arena",
+					"effect_type": "vega_ua_guarantee",
```

Comment:
I wonder if there's a way to implement this (and the regular UA) with character_bonus instead? probably not a big deal though